diff --git a/node_modules/@sentry/react/types/errorboundary.d.ts b/node_modules/@sentry/react/types/errorboundary.d.ts
index 4e1f326..779e0e7 100644
--- a/node_modules/@sentry/react/types/errorboundary.d.ts
+++ b/node_modules/@sentry/react/types/errorboundary.d.ts
@@ -9,6 +9,7 @@ export declare type FallbackRender = (errorData: {
     resetError(): void;
 }) => React.ReactElement;
 export declare type ErrorBoundaryProps = {
+    children: React.ReactNode;
     /** If a Sentry report dialog should be rendered on error */
     showDialog?: boolean;
     /**
